forge-gui/res/cardsfolder/a/ajani_adversary_of_tyrants.txt

```diff
@@ -4,7 +4,7 @@ Types:Legendary Planeswalker Ajani
 Loyalty:4
 A:AB$ PutCounter | Cost$ AddCounter<1/LOYALTY> | Planeswalker$ True | CounterType$ P1P1 | CounterNum$ 1 | TargetMin$ 0 | TargetMax$ 2 | ValidTgts$ Creature | TgtPrompt$ Select target creature | SpellDescription$ Put a +1/+1 counter on each of up to two target creatures.
 A:AB$ ChangeZone | Cost$ SubCounter<2/LOYALTY> | Planeswalker$ True | Origin$ Graveyard | Destination$ Battlefield | ValidTgts$ Card.Creature+cmcLE2+YouCtrl | TgtPrompt$ Select target creature card with mana value 2 or less | SpellDescription$ Return target creature card with mana value 2 or less from your graveyard to the battlefield.
-A:AB$ Effect | Cost$ SubCounter<7/LOYALTY> | Planeswalker$ True | Ultimate$ True | Name$ Emblem - Ajani, Adversary of Tyrants | Image$ emblem_ajani_adversary_of_tyrants | Triggers$ EffPhase | Duration$ Permanent | SpellDescription$ You get an emblem with "At the beginning of your end step, create three 1/1 white Cat creature tokens with lifelink."
+A:AB$ Effect | Cost$ SubCounter<7/LOYALTY> | Planeswalker$ True | Ultimate$ True | Name$ Emblem — Ajani, Adversary of Tyrants | Image$ emblem_ajani_adversary_of_tyrants | Triggers$ EffPhase | Duration$ Permanent | SpellDescription$ You get an emblem with "At the beginning of your end step, create three 1/1 white Cat creature tokens with lifelink."
```

Comment:
The change is slightly problematic for emblems, a few code parts expect the short one :/

Reply:
Yea code can be changed
Just don‘t have the time at this point

Reply:
Here's a start on my side, FWIW: https://github.com/Card-Forge/forge/pull/5536